packages/plugin-expo-device/device.js

```diff
@@ -2,6 +2,10 @@ const Device = require('expo-device')
 const Constants = require('expo-constants').default
 const { Dimensions, Platform } = require('react-native')
 const rnVersion = require('react-native/package.json').version
+const cuid = require('@bugsnag/cuid')
+const SecureStore = require('expo-secure-store')
+
+const DEVICE_ID_KEY = 'bugsnag-anonymous-id'
```

Comment:
Is there a reason not to call this `bugsnag-device-id`? I'm not sure if the name is at all visible externally though...